types/api/mongo-query.ts

```diff
@@ -1,8 +1,8 @@
 export interface MongoQuery {
 
-    count : boolean
-    length: Number
-    skip  : Number
-    sort  : String
+    count? : boolean
+    length?: Number
+    skip?  : Number
```

Comment:
should be `number` no caps

> number (lowercase): This is the primitive type in TypeScript (and JavaScript) used to represent numeric values.
> Number (uppercase): This refers to the built-in JavaScript Number object constructor.


```suggestion
    length?: number
    skip?  : number
```